ormin/ormin_postgre.nim

```diff
@@ -95,12 +98,17 @@ template bindResult*(db: DbConn; s: PStmt; idx: int; dest: bool;
                      t: typedesc; name: string) =
   dest = isTrue(pqgetvalue(queryResult, queryI, idx.cint))
 
+
 proc fillString(dest: var string; src: cstring; srcLen: int) {.inline.} =
-  var i = 0
-  while src[i] != '\0':
-    dest.add src[i]
-    inc i
-    
+  var
+    c = src[0]
+    i = 0
+  while c != '\0':
+    dest.add($c)
+    i = i + 1
```

Comment:
what? I think the old ``fillString`` was fine.

Reply:
To be honest I don't remember anymore as it was a while ago, but if is I did it, it was for a reason. I'll provide more info tomorrow.

Reply:
Confirmed, old fillString works fine now, I think is due to some changes in Nim itself. And I see that some other issues that i had previously are also working now with Nim 1.0.0. So I'll create Issues with the specific findings and then PRs with the fixes, so feel free to reject this PR if that's OK with you.